src/components/generic/Button/Button.scss

```diff
@@ -75,7 +75,7 @@
   border: var(--wxc-button--ghost--normal--border);
   border-radius: 50%;
   color: var(--wxc-button--ghost--normal--color);
-  padding: 0.5625rem;
+  padding: 0.375rem;
```

Comment:
Why are you changing the padding? doesn't this affect all buttons?

Reply:
Hi @lalli-flores 

This change does not affect all the buttons, just the ghost button to look similar to the Figma design.

Thank you.